scripts/createYOEOLabelsFromTORSO-21.py

```diff
@@ -122,8 +125,8 @@ def range_limited_float_type_0_to_1(arg):
             # Skip annotations, if is not a bounding box or should be skipped or is blurred or concealed and user chooses to skip them
             if (annotation['type'] in CLASSES['segmentation_classes'] or  # Handled by segmentations
                 annotation['type'] in CLASSES['skip_classes'] or  # Skip this annotation class
-                (args.skip_blurred and annotation['blurred']) or
-                (args.skip_concealed and annotation['concealed'])):
+                (args.skip_blurred and hasattr(annotation, 'blurred') and annotation['blurred']) or
```

Comment:
One could also do `annotation.get('blurred', False)` which would be more concise.